data/criteo.txt

```diff
@@ -0,0 +1,1000 @@
+1	5	110		16		1	0	14	7	1		306		62770d79	e21f5d58	afea442f	945c7fcf	38b02748	6fcd6dcb	3580aa21	28808903	46dedfa6	2e027dc1	0c7c4231	95981d1f	00c5ffb7	be4ee537	8a0b74cc	4cdc3efa	d20856aa	b8170bba	9512c20b	c38e2f28	14f65a5d	25b1b089	d7c1fc0b	7caf609c	30436bfc	ed10571d
```

Comment:
Why do we have four different files with the same data?
Only one seems to be necessary.